vignettes/savvyr_example.Rmd

```diff
@@ -0,0 +1,115 @@
+---
+title: "SAVVY: estimation of AE probabilities"
+package: savvyr
+output:
+  rmarkdown::html_vignette:
+          toc: true
+vignette: |
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{SAVVY: estimation of AE probabilities}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options:
+  chunk_output_type: console
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+```{r, include=TRUE, echo=TRUE}
+# packages
+packs <- c("data.table", "etm", "survival", "mvna", "knitr", "kableExtra")
+for (i in 1:length(packs)) {
+  library(packs[i], character.only = TRUE)
+}
+```
```

Comment:
we should not need this, I would only directly load packages as needed below (only `savvyr` load on top here)

Reply:
adapted